README.md

```diff
@@ -1,68 +1,79 @@
+## Cloud platforms examples
 
-## Custom deployment
+Platform-native solutions sorted by cloud service provider:
 
-To customize your deployment, you can use containers as a service or create your own advanced deployment with the examples below as your base.
+### Microsoft Azure
 
-### Before you begin
+- [Azure Container Apps using ARM (Azure Resource Manager) template](https://support.1password.com/scim-deploy-azure/)
+- [Azure Kubernetes Service using Cloud Shell](https://support.1password.com/cs/scim-deploy-azure-kubernetes/)
 
-Before you begin deploying 1Password SCIM Bridge, review the [Preparation Guide](/PREPARATION.md). The guide will help you plan for some of the technical components of the deployment and consider some issues you may encounter along the way.
 
-### Containers as a service deployment options
+### Amazon Web Services (AWS)
+- ✨ **BETA** [Amazon ECS using CloudFormation](/beta/aws-ecsfargate-cfn)
+- [Amazon ECS using Terraform](/aws-ecsfargate-terraform)
 
-Containers as a service (CaaS) can simplify your deployment by using the built-in tools of the CaaS for DNS and certificate management. This gives you an easy, low-cost SCIM bridge with minimal infrastructure management requirements.
+### Google Cloud (GCP)
 
-- Azure Container Apps
-  - [Azure Portal Deployment using an ARM Template](https://support.1password.com/scim-deploy-azure/) - recommended for most Azure Container App deployments
-  - [CLI Deployment](https://support.1password.com/cs/scim-deploy-azure-container-apps/) - recommended if you want to use Azure CLI
-  - [Azure Portal Deployment (Manual)](/azure-container-apps/README.md) - recommended for those not able to use the tools in the above CLI deployment guide
-  - [Container Apps advanced customizations](/azure-container-apps/ADVANCED.md)
-- DigitalOcean App Platform
-  - [DigitalOcean Portal Deployment](https://support.1password.com/cs/scim-deploy-digitalocean-ap/)
-  - [DigitalOcean command line tool & the 1Password CLI Deployment](/do-app-platform-op-cli)
+- [Cloud Run using Cloud Shell](/beta/google-cloud-run)
+- [Google Kubernetes Engine using Google Cloud Marketplace](https://support.1password.com/scim-deploy-gcp/)
 
-### Advanced deployment options
+### DigitalOcean
 
-If you have particular requirements for your environment, we recommend an advanced deployment. These example configurations will give you a base to create the deployment from, as well as explain what 1Password SCIM Bridge needs to function and how to maintain your bridge once you've deployed it.
+- [App Platform using DigitalOcean console](https://support.1password.com/cs/scim-deploy-digitalocean-ap/)
+- [DigitalOcean Kubernetes using DigitalOcean Marketplace](https://support.1password.com/scim-deploy-digitalocean/)
 
-- [AWS ECS Fargate with Terraform](/aws-ecsfargate-terraform)
-- [Azure Kubernetes Service](https://support.1password.com/cs/scim-deploy-azure-kubernetes)
-- [Docker Swarm](/docker)
+## Generic examples
```

Comment:
Nitpick: Consider replacing "Generic examples" with "Platform-agnostic examples" or "Self-hosted examples options". A more descriptive term in line with the "Cloud platform examples" heading, that describes the Docker, Kubernetes, and Helm Chart examples.
```suggestion
## Platform-agnostic examples
```